### ISSUE TITLE
support delegating field by map

### ISSUE THREAD

**Issue by CXwudi** [wuseal/JsonToKotlinClass#344]
Is it possible to support https://kotlinlang.org/docs/delegated-properties.html#storing-properties-in-a-map

**Comment by wuseal** [comment]
Yes, It's possible with Extension  https://github.com/wuseal/JsonToKotlinClass/blob/master/doc_for_extensions.md
